kronolith/lib/Application.php

```diff
@@ -945,6 +945,35 @@ public function davGetCollections($user)
                 '{http://sabredav.org/ns}read-only' => !$share->hasPermission($hordeUser, Horde_Perms::EDIT),
             );
         }
+        // External Calendars from other Horde Apps
+                foreach ($calendar_manager->get(Kronolith::ALL_EXTERNAL_CALENDARS) as $calendarId => $calendar) {
+            // We don't want to duplicate tasks handling
+            if ($calendar->api() == 'tasks') {
+                continue;
+            }
+            try {
+                $id = $dav->getExternalCollectionId($calendar->internalId(), 'calendar');
+            } catch (Horde_Dav_Exception $e) {
+            }
+            $calendars[] = array(
+                'id' => $id,
+                'uri' => $id,
+                '{' . CalDAV\Plugin::NS_CALENDARSERVER . '}shared-url' =>
+                    $calendar->caldavUrl(),
+                'principaluri' => 'principals/' . $user,
+                '{http://sabredav.org/ns}owner-principal' =>
+                    'principals/-system-', // TODO: Not sure who should really own these, system or the calling user
+                '{DAV:}displayname' => sprintf("%s: %s", ucfirst($calendar->api()),  $calendar->name()),
+                '{' . CalDAV\Plugin::NS_CALDAV . '}calendar-description' =>
```

Comment:
I would actually leave this off until there is a better value to get than the duplication of the calendar name.